plugins/woocommerce/tests/e2e-pw/tests/shopper/my-account-pay-order.spec.js

```diff
@@ -21,14 +28,18 @@ test.describe( 'Customer can pay for their order through My Account', () => {
 			.then( ( response ) => {
 				productId = response.data.id;
 			} );
+		// create customer
+		await api
+			.post( 'customers', customer )
+			.then( ( response ) => ( customer.id = response.data.id ) );
```

Comment:
This test is much more stable when it uses its own test customer rather than using the customer account saved in the smoke test site.